src/movies.ts

```diff
@@ -109,8 +118,10 @@ function updateMovieGenre(
   newGenre: string
 ): Movie[] {
   // write your code here...
-
-  return []; // replace empty array with what you see is fit
+  movies.forEach((movie) =>
+    movie.title === title ? (movie.genre = newGenre) : movie
+  );
+  return movies;
```

Comment:
This could have been solved using `.find`:

```ts
function updateMovieGenre(
  movies: Movie[],
  title: string,
  newGenre: string
): Movie[] {
  const movieToUpdate = movies.find((movie) => movie.title === title) // returns the movie or undefined
  if(movieToUpdate)
    movieToUpdate.genre = newGenre;
  return movies;
}
```